---
date: 2025-09-28
title: Atualização (ALRA)
layout: default
---
Erro: Problema a conectar a alra.pt
HTTPConnectionPool(host='base.alra.pt', port=82): Max retries exceeded with url: /4DACTION/w_recebe_pesquisa_audi/1%20method=?wformvalida2.x=15&wformvalida2.y=21&w_legis=XIII&w_numero_audi=&w_numero_audi_fim=&w_entrada_audi=&w_entrada_audi_fim=&w_d_entrada_audi=&w_d_entrada_audi_fim=&w_processo_audi=&w_titulo_audi=&POPcomissao=&w_assunto_audi=&w_d_envio_audi=&w_d_envio_audi_fim=&w_d_governo_audi=&w_d_governo_audi_fim=&w_d_apre1_audi=&w_d_apre1_audi_fim=&w_d_limite_audi=&w_d_limite_audi_fim=&w_d_resp_audi=&w_d_resp_audi_fim=&w_d_prorro_audi=&w_d_prorro_audi_fim=&w_d_envioar_audi=&w_d_envioar_audi_fim=&w_d_apre2_audi=&w_d_apre2_audi_fim= (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f813d4eecf0>, 'Connection to base.alra.pt timed out. (connect timeout=120)'))
.
